src/main/java/com/hcl/appscan/jenkins/plugin/scanners/Scanner.java

```diff
@@ -43,15 +44,15 @@ public String getTarget() {
 	protected String resolvePath(String path, VariableResolver<String> resolver) {
 		//First replace any variables in the path
 		path = Util.replaceMacro(path, resolver);
-		
+		Pattern pattern = Pattern.compile("^(\\\\|/|[a-zA-Z]:\\\\)");
+
 		//If the path is not absolute, make it relative to the workspace
-		File file = new File(path);
-		if(!file.isAbsolute()) {
-			String targetPath = "${WORKSPACE}" + File.separator + file.getPath();
+		if(!(path.equals("")) && ((!(path.startsWith("/"))) && (!(path.startsWith("\\"))) && (!pattern.matcher(path).find()))){
+			String targetPath = "${WORKSPACE}" + "//" + path ;
```

Comment:
The regex should handle all of the cases, so we don't need all of the additional checks in the "if" statement. Also, we only need a single forward slash (i.e. "/"), not the double forward slash.